spec/system/distribution_system_spec.rb

```diff
@@ -308,7 +314,7 @@
 
   # TODO: This should probably be in the Request resource specs, not Distribution
   context "When creating a distrubition from a request" do
-    before do
+    it "it sets the distribution id and fulfilled status on the request" do
```

Comment:
🥇 Really like this change. `before` blocks tend to decrease readability of specs IMO. Especially when they are really only needed for one spec.

Side note: does it read oddly to you to have the test read `it "it does a thing"`?

Reply:
LOL I totally overlooked that. Yes it reads oddly and no that was not intentional. Good eyes